android/src/main/java/com/pinmi/react/printer/adapter/BLEPrinterAdapter.java

```diff
@@ -188,9 +176,53 @@ public void run() {
     @Override
     public void printByteData(byte[] rawBase64Data, Callback errorCallback) {
     }
-
+    
+    @Override
+    public void connectAndSend(String host, Integer port, String rawBase64Data, Callback successCallback, Callback errorCallback) {
+    }
 
     @Override
-    public void connectAndSend(String host, Integer port, String data, Callback successCallback, Callback errorCallback) {
+    public void connectAndSend(PrinterDeviceId printerDeviceId, String data, Callback successCallback, Callback errorCallback) {
```

Comment:
Same function names being declared ?  Above one is for network and this one for any other use case ?

Or by brand ?

Reply:
Yes. function overriding. for net and bluetooth printing.